app/services/cloud_front_header_parser.rb

```diff
@@ -10,6 +10,6 @@ def client_port
 
   # Source IP and port for client connection to CloudFront
   def viewer_address
-    @request.get_header 'CloudFront-Viewer-Address'
+    @request.get_header 'HTTP_CLOUDFRONT_VIEWER_ADDRESS'
```

Comment:
I'd be curious what happens if we do
```suggestion
    @request.headers['CloudFront-Viewer-Address']
```
based on these docs: https://api.rubyonrails.org/v2.3/classes/ActionController/Request.html#M001828